android/hybridcommon/build.gradle.kts

```diff
@@ -0,0 +1,85 @@
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
+plugins {
+    alias(libs.plugins.androidLibrary)
+    alias(libs.plugins.kotlinAndroid)
+    alias(libs.plugins.androidJunit5)
+    alias(libs.plugins.mavenPublish)
+    alias(libs.plugins.detekt)
+}
+
+dependencies {
+    detektPlugins(libs.detekt.formatting)
+}
+
+detekt {
+    buildUponDefaultConfig = true
+    baseline = file("config/detekt/detekt-baseline.xml")
+}
+
+android {
+    namespace = "com.revenuecat.purchases.hybridcommon"
+    compileSdk = 34
```

Comment:
This is being updated from 33, not sure if it can cause any issue (I don't think so)

Reply:
Yeah, it was part of adding the new module. I believe it should be fine after checking the changes. Thanks for checking!